backend/tests/integration_docker/test_computed_attributes.py

```diff
@@ -190,3 +203,47 @@ async def test_transform_based_computed_attribute(self, client: InfrahubClient,
             await sleep(1)
 
         assert sth_router_1_swe.name.value == swe_name_router_1
+
+    async def test_update_schema_not_related_to_computed_attribute(
+        self, client: InfrahubClient, schema_computed_tshirt: dict
+    ) -> None:
+        nbr_task_before = await client.task.count(
+            filters=TaskFilter(workflow=[COMPUTED_ATTRIBUTE_JINJA2_UPDATE_VALUE.name])
+        )
+
+        # Update schema LocationSite with a change that IS NOT related to the computed attribute
+        # The computed attribute of type Jinja2 should not be updated neither on the sites nor on the continents
+        schema_computed_tshirt["nodes"][4]["description"] = "New Description that will trigger a new schema"
+        tshirt_schema = await client.schema.load(schemas=[schema_computed_tshirt], wait_until_converged=True)
+        assert tshirt_schema.schema_updated
+
+        # Validate that the schema is in sync after loading the device and interface schema
+        assert await client.schema.in_sync()
+
+        await sleep(1)
+        await wait_for_all_tasks_to_be_completed(client)
+
+        nbr_task_after_not_related = await client.task.count(
+            filters=TaskFilter(workflow=[COMPUTED_ATTRIBUTE_JINJA2_UPDATE_VALUE.name])
+        )
+        assert nbr_task_after_not_related == nbr_task_before
+
+        # Update schema LocationSite with a change that IS related to the computed attribute
+        # The computed attribute of type Jinja2 should be updated
+        schema_computed_tshirt["nodes"][4]["attributes"][3]["computed_attribute"]["jinja2_template"] = (
+            "WELCOME TO {{ name__value }}!"
+        )
+        tshirt_schema = await client.schema.load(schemas=[schema_computed_tshirt], wait_until_converged=True)
+        assert tshirt_schema.schema_updated
+
+        # Validate that the schema is in sync after loading the device and interface schema
+        assert await client.schema.in_sync()
+
+        await sleep(1)
```

Comment:
As we are waiting within `wait_for_all_tasks_to_be_completed`, is this sleep here and the similar one above just to ensure that the tasks themselves have had time to start?